CustomLocaleHandler.inc.php

```diff
@@ -22,7 +22,12 @@ function printCustomLocaleChanges($args, $request) {
 		$contextId = $context->getId();
 
 		$publicFilesDir = Config::getVar('files', 'public_files_dir');
-		$customLocaleDir = "$publicFilesDir/presses/$contextId/" . CUSTOM_LOCALE_DIR;
+		/* UZH CHANGE OJS-71 2019/03/22/mb */
+		/* $customLocaleDir = "$publicFilesDir/presses/$contextId/" . CUSTOM_LOCALE_DIR; */
+		$fileDirectories = Application::getFileDirectories();
```

Comment:
I would suggest using `PublicFileManager::getContextFilesPath` instead of `Application::getFileDirectories`.

Reply:
Anything that makes it easier to obtain the path is welcome - I'm just starting to dive into OJS programming. Is this function also available in OMP?

Reply:
OK, sorry - I have found it in OMP, too. Will implement it.

Reply:
See https://github.com/mpbraendle/customLocale/commit/a607c97e06f9cccadc93def056fb8763632c4a67